src/components/home.css

```diff
@@ -1,3 +1,108 @@
+body {
+  background: #f7f5f5;
+  font-family: sans-serif;
+}
+
 .sec {
-  border: 1px solid black;
+  padding: 12px 12px;
+  background: #fff;
+  margin-bottom: 12px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.category {
+  font-size: 10px;
+  font-weight: 600;
+  color: #555454;
+}
+
+.title {
+  font-size: 16px;
+  font-weight: 600;
+  margin: 2px 0;
+  width: auto;
+  overflow: auto;
+}
+
+.title::-webkit-scrollbar {
+  display: none;
+}
+
+.author {
+  font-size: 11px;
+  color: #9191ee;
+}
+
+.btnDiv {
+  display: flex;
+  gap: 10px;
+}
+
+.button {
+  background: none;
+  color: #9191ee;
+  font-size: 10px;
+  border: none;
+}
+
+.button:nth-child(1) {
+  border-right: 1px solid black;
+  padding-right: 7px;
```

Comment:
- [ ] Avoid using the Same Rule Repetitively as you have done in lines 52 and 57(padding-right: 7px;). This is called CSS bad practices. To avoid this, you can use a class instead.